bin/modify_alarms.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script to modify the billing alarms for TheBoss changing them to have a period of 60s"""
```

Comment:
Some whitespace separating this from the copyright would be helpful. I totally missed it the first time through. Also, does this _modify_ existing alarms? The help output states that this should only be run once.

Reply:
Fixed.  I am also going to move this to user-scratch.  It was  used to modify how often the alarms went off, this ended up not working.  The alarms need to stay at 6 hours otherwise the reset and fire every 6 hours.  It may still be handy if we ever want to update something with these alarms again.